### ISSUE TITLE
Remove hh:mm from enlisted time

### ISSUE THREAD

**Issue by TheMasterDingo** [ghost-squadron/calypso#4]
Looks like it’s a placeholder clock.

**Comment by hutli** [comment]
**TLDR:** We can either get localisation on the date or remove the annoying timestamp, and I've currently prioritised localisation.

I've made many considerations regarding this.

I am currently using the Timestamp field in the Discord Embed. The Timestamp field, unfortunately, only allows for ISO8601 timestamps requiring you to include a specific time (not just a date). You can technically exclude the time in ISO8601, but this just means "convert to 00:00:00 local time". I've tried to send the request directly without `discord.py`, but even if I exclude the time and timezone, it automatically converts to local time 00:00:00 (as per the ISO standard). I have also tested UTC 00:00:00 and local time 00:00:00, but no matter what, Discord always displays the localised time in the message.

The timestamp is the only correctly typed field for "time" in Discord Embeds. Alternatively, using a plain string somewhere else is possible, but that would mean losing localisation on the date, which I currently value higher.
